src/unity/lib/visualization/CMakeLists.txt

```diff
@@ -14,10 +14,13 @@ make_library(
     process_wrapper.cpp
     scatter.cpp
     show.cpp
+    summary_view.cpp
     thread.cpp
     transformation.cpp
     vega_data.cpp
     vega_spec.cpp
+
+    plot/plot.cpp
```

Comment:
Since there isn't anything else in the `plot/` directory, we probably don't need a `plot` directory yet. I'd move `plot.[h|c]pp` up into the parent directory.

Reply:
got it